content/php-client/resources/target-market-settings/measurement-families.md

```diff
@@ -0,0 +1,116 @@
+### Measurement family
+
+#### Get a list of measurement families
+::: php-client-availability versions=6.0 editions=CE,EE
```

Comment:
```suggestion
::: php-client-availability versions=5.0 editions=CE,EE
```
Where does the 6 come from ?

Reply:
From the future !
Joke aside, we need to release the v6.0 of the client for including the new measurement family endpoints. Seen with @NolwennP

Reply:
Yup, those new endpoints will be released in a 6.0 version of the PHP client.